k8s/namespaces/ccd/message-publisher/aat.yaml

```diff
@@ -13,10 +13,3 @@ spec:
       environment:
         DATA_STORE_DB_HOST: ccd-data-store-api-postgres-db-v11-aat.postgres.database.azure.com
         DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-aat
```

Comment:
Can you remove `DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-v11-aat` as it comes from boostrap.yaml